app/src/main/java/io/intrepid/thirdpartylibraryexcercise/MainActivity.java

```diff
@@ -1,17 +1,29 @@
 package io.intrepid.thirdpartylibraryexcercise;
 
 import android.app.ProgressDialog;
+import android.content.Context;
 import android.graphics.Bitmap;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
 import android.view.View;
 import android.widget.Button;
 import android.widget.ImageView;
+import android.widget.Toast;
 
-public class MainActivity extends AppCompatActivity implements MainContract.View, DownloadCatImageTask.Callback {
+import com.squareup.picasso.Picasso;
 
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+public class MainActivity extends AppCompatActivity implements MainContract.View {
+
+    @BindView(R.id.image_view)
     ImageView imageView;
+
+    @BindView(R.id.new_cat_button)
```

Comment:
You actually don't need to do `@BindView` on this, since you're already using `@OnClick` with this id. If we had to do anything else to the button, then we would have to do this, but since we don't have any other references to this button, we can actually just remove this variable